src/pages/gettingstarted/angular.hbs

```diff
@@ -0,0 +1,368 @@
+          </span>
+          .
+        </p>
+      </li>
+      <li class="sprk-u-mvm">
+        <p>
+          Install additional peer dependencies:
+        </p>
+        <ul class="drizzle-b-List sprk-u-mbm">
+          <li class="sprk-u-mvm">
+            <div class="drizzle-c-Command">
+              <code class="drizzle-CodeHighlight">
+                npm install --save-dev tiny-date-picker</code>
+            </div>
+          </li>
+          <li class="sprk-u-mvm">
+            <div class="drizzle-c-Command">
+              <code class="drizzle-CodeHighlight">
+                npm install --save-dev lodash</code>
+            </div>
+          </li>
+        </ul>
+      </li>
+      <li class="sprk-u-mvm">
+        <p>
+          Import the Sass setup file in your Angular application's global styles Sass file.
+          This can be done in the scss file specified
+          at the highest level of your app.
+          This will bring all Spark-Core patterns into your build. The .scss file extension
+          is needed in the case of Angular so as to not conflict with JS files.
+        </p>
+        <div class="drizzle-c-Command">
+          <code class="drizzle-CodeHighlight">
+            @import &quot;node_modules/@sparkdesignsystem/spark-core/spark-core.scss&quot;;
+          </code>
+        </div>
+      </li>
+      <li>Your HTML element needs to have the following class:
+        <div class="drizzle-c-Command">
+          <code class="drizzle-CodeHighlight">
+            sprk-u-JavaScript
+          </code>
+        </div>
+      </li>
+      <li class="sprk-u-mvm">
+        <p>
+          Add the following in your main app.module.ts file:
+          <div class="drizzle-c-Command">
+            <code class="drizzle-CodeHighlight">
+              import { SparkCoreAngularModule } from "@sparkdesignsystem/spark-core-angular";
+            </code>
+          </div>
+        </p>
+        <p>
+          Add the 
+          <span class="drizzle-c-Command">
+            <code class="drizzle-CodeHighlight">
+              SparkCoreAngularModule
+            </code>
+          </span> 
+          to the NgModule imports array.
+        </p>
+
+      </li>
+      <li class="sprk-u-mvm">
+        <p>
+          Add the following in your main app.module.ts file:
+          <div class="drizzle-c-Command">
+            <code class="drizzle-CodeHighlight">
+              import { BrowserAnimationsModule } from '@angular/platform-browser/animations';
+            </code>
+          </div>
+        </p>
+        <p>
+          Add the 
+            <span class="drizzle-c-Command">
+              <code class="drizzle-CodeHighlight">
+                BrowserAnimationsModule
+              </code>
+            </span>
+          to the NgModule imports array.
+        </p>
+      </li>
+      <li class="sprk-u-mvm">
+        <p>Spark does not provide icons directly. What you see below are proprietary icons in use by Quicken Loans. To supply
```

Comment:
can you change it from see below to like 'see on our icons page'?

Reply:
Updated